content/en/docs/reference/glossary/replication-controller.md

```diff
@@ -1,19 +1,28 @@
 ---
-title: Replication Controller
+title: ReplicationController
 id: replication-controller
 date: 2018-04-12
 full_link: 
 short_description: >
-  Kubernetes service that ensures a specific number of instances of a pod are always running.
+  Ensures a specific number of instances of a Pod are running.
```

Comment:
@sftim , some general impressions:

Line 7, the short description does not tell the reader whether the ReplicationController is a resource, service, or ...
Line 14, I don't think you need to highlight (in parens) that this resource is deprecated. Just mention that in the note.
Line 19, A ReplicationController automatically adds or removes 
OR
ReplicationController automatically adds or removes

... `based on a set value` is a bit vague. I think the original phrasing is explicit.
Line 20, You introduce the control plane instead of the ReplicationController:
`The control plane ensures that the defined number of ...`
Lines 25 -27. Can the note simply say,
ReplicationController is deprecated.
When was the ReplicationController deprecated? Hopefully this information and the suggested replacement for ReplicationController is listed on the actual concept page(s).
The ReplicationController concept page lists this note as the first item on the page,
https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/

Reply:
I've made some changes.

I'm not sure when ReplicationController was deprecated. If that's important to add, I can log a separate issue to track looking that up and documenting it.